vcpkg.json

```diff
@@ -1,12 +1,28 @@
 {
-    "name": "openloco",
-    "version-string": "22.02",
-    "dependencies": [
-        "benchmark",
-        "breakpad",
-        "gtest",
-        "libpng",
-        "openal-soft",
-        "sdl2"
-    ]
+  "name": "openloco",
+  "version-string": "22.02",
+  "dependencies": [
+    "benchmark",
+    {
+      "name": "breakpad",
+      "platform": "windows"
+    },
+    "gtest",
+    "libpng",
+    "openal-soft",
+    {
+      "name": "sdl2",
+      "default-features": false,
+      "features": [
+        "x11"
+      ]
+    }
+  ],
+  "builtin-baseline": "300239058e33420acd153135b3f6e6b187828992",
+  "overrides": [
+    {
+        "name": "openal-soft",
+        "version": "1.24.1"
+    }
+]
```

Comment:
This works around a linker issue when statically linking openal-soft, as mentioned in #2882. For the time being I'm just locking to the previous version of openal-soft. They've already fixed this by namespacing their vendored `fmt` differently so that it won't clash, but they've not made a release since then. This can be removed entirely when openal-soft releases their fix and vcpkg is updated with it.